### ISSUE TITLE
Bad redirection after successful admin login in subdirectory multisite 

### ISSUE THREAD

**Issue by jza34** [getgrav/grav-plugin-admin#1487]
I have setup subdirectory multisite and it works well!
But when I login in the subsite admin I am redirected to the main URL admin for login...
If I go back to the subsite admin URL again and I am logged!

Example:
- go to /example.com/mysubsite/admin
- login
- you are redirected to /example.com/admin
- manualy go to /example.com/mysubsite/admin without login needed

I don't have this behaviour in a subdomain multisite configuration

Thanks for help!

**Comment by masetto** [comment]
Hello,
I have the same question. Any news about it?
Thanks

**Comment by noirb** [comment]
I have the same issue (Admin panel version 1.8.15, Login version 2.8.2, on both Grav 1.5.6 and 1.6.0-beta.7).

I tried changing the `Administrator path` setting to include the subsite (e.g. `/subsite/admin`), but it didn't help (after login I was still redirected to `example.com/admin`, and needed to go to `example.com/subsite/subsite/admin` to access the subsite's admin panel). This is the behavior I would expect, though.

I also tried setting the `Redirect after login` field in the Login plugin to `/subsite/admin`, but this field appears to be ignored when logging directly into the admin panel and I was still redirected to `example.com/admin`.

I can't tell if the issue is the Login plugin redirecting incorrectly or the Admin plugin providing the incorrect URL, but since this issue already exists here I thought it would be best not to create a duplicate in the Login repo :)
